services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -150,9 +151,16 @@ testsPublic s =
             [ test s "happy flow" testInWhitelist,
               test s "handshake between LH device and user with old clients is blocked" testOldClientsBlockDeviceHandshake,
               testGroup "no-consent" $
-                flip fmap [(a, b, c, d) | a <- [minBound ..], b <- [minBound ..], c <- [minBound ..], d <- [minBound ..]] $
-                  \args@(a, b, c, d) ->
-                    test s (show args) $ testNoConsentBlockOne2OneConv a b c d,
+                ( ( (,,,)
+                      <$> (("connectFirst" :: String,) <$> [False, True])
+                      <*> (("teamPeer" :: String,) <$> [False, True])
+                      <*> (("approveLH" :: String,) <$> [False, True])
+                      <*> (("testPendingConnection" :: String,) <$> [False, True])
+                  )
```

Comment:
Nice! You can also use `do` notation in the list monad to improve readability here:
```haskell
do connectFirst <- ("connectFirst",) <$> [False, True]
   ...
   let name = ...
   pure $ test s name $ ...
```

Reply:
Ah, right, done